cursorless-talon/src/apps/vscode_settings.py

```diff
@@ -61,7 +61,6 @@ def vscode_get_setting_with_fallback(
             return actions.user.vscode_get_setting(key, default_value), False
         except Exception:
             print(fallback_message)
-            traceback.print_exc()
```

Comment:
This just created a lot of noise in the Talon log. We know were the problem is without this.